src/Constants.ts

```diff
@@ -83,4 +85,16 @@ export enum Severity {
 	LOW = 3,
 	MODERATE = 2,
 	HIGH = 1
-}
\ No newline at end of file
+}
+
+export const FILE_EXTS_TO_LANGUAGE: Map<string, string> = new Map([
+	// apex
+	['.cls', 'apex'],
+	['.trigger', 'apex'],
+	// vf
+	['.component', 'vf'],
+	['.page', 'vf'],
```

Comment:
Can we use constants for the language value so that we don't accidentally have typos or mixed case?